velox/docs/configs.rst

```diff
@@ -108,7 +108,13 @@ Expression Evaluation Configuration
    * - cast_to_int_by_truncate
      - bool
      - false
-     - This flags forces the cast from float/double to integer to be performed by truncating the decimal part instead of rounding.
+     - This flag forces the cast from float/double to integer to be performed by truncating the decimal part instead of rounding.
```

Comment:
drop 'This flag' and use active verbs for ease of reading

```
If set, cast from float or double to integer truncates the decimal part., otherwise rounds.
```